food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```
@@ -94,8 +109,14 @@ public String postTruckReview(@AuthenticationPrincipal User user, @PathVariable
 
     @Secured({"ROLE_OWNER"})
     @PostMapping("/truck/create")
-    public Truck createTruck(@AuthenticationPrincipal User u, @RequestBody CreateTruckParams data) {
-        return truckService.createTruck(u.getId(), data.truckName);
+    public Truck createTruck(@AuthenticationPrincipal User u, @RequestBody CreateTruckParams data){
+        Truck truck = truckService.createTruck(u.getId(), data.truckName);
+        try {
+            indexingService.initiateIndexing();
+        }catch (InterruptedException e){
+            return truck;
```

You don't technically need this return here. The catch would cause it to handle this code, then continue execution outside the catch, where it's returning truck anyway.

Also, maybe we should at least log the exception instead of swallowing it.

---

Not sure why it doesn't show up in the review files, but I commented this before and he added logging statements.
https://github.com/Spring-Kings/food-truck-finder/pull/16/commits/b43914e05d1e70b499f3aea05a8ef5d760421fb8